WDAC-Policy-Wizard/app/MSIX/AppxManifest.xml

```diff
@@ -4,7 +4,7 @@
   xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
   xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities">
   <Identity Name="Microsoft.WDAC.WDACWizard"
-            Version="1.6.6.0"
+            Version="1.6.6.1"
```

Comment:
nitpick, first number for API/ABI breaks. Second number for features. Third number for bug fixes. Fourth number for release candidates.